config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
+  get 'user/new'
+
   root 'home#index'
   resources :cards
+  resources :users
+  resources :user_sessions
+
+  get 'login' => 'user_sessions#new', :as => :login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.